airbyte-integrations/connectors/destination-langchain/destination_langchain/config.py

```diff
@@ -135,10 +136,16 @@ def resolve_refs(schema: dict) -> dict:
         pyschema: dict = json.loads(str_schema)
         del pyschema["definitions"]
         return pyschema
+    
+    @staticmethod
+    def remove_discriminator(schema: dict) -> None:
+        """pydantic adds "discriminator" to the schema for oneOfs, which is not treated right by the platform as we inline all references"""
```

Comment:
change looks good. leaving a note that this is something we could do at the CDK level if this is a widespread issue